server/impl/src/main/java/com/walmartlabs/concord/server/org/secret/SecretUpdateRequest.java

```diff
@@ -44,6 +44,9 @@
     @ConcordKey
     String name();
 
+    @Nullable
+    UUID ownerId();
```

Comment:
Also, as @brig pointed out, other entities use `Owner` structure. I think we should use it here too, for symmetry.

Reply:
missed this. I'm on it..